### PR TITLE
[ACR] Rename credential_type to grant_type

### PR DESCRIPTION
**Pull request by DavidObando**

### CODE REVIEW

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py [hunk 1]
```diff
@@ -45,15 +45,15 @@ def _get_login_token(login_server, only_refresh_token=True, repository=None):
...
         content = {
+            'grant_type': 'refresh_token',
             'service': params['service'],
-            'credential_type': 'user',
             'tenant': tenant,
             'refresh_token': refresh
         }
     else:
         content = {
+            'grant_type': 'spn',
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py]: LGTM. NIT: seems a bit odd to use `spn` with `grant_type`?

Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py]: This actually makes sense. I was thinking about different terms in AAD and OAuth2.

Comment (DavidObando) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py]: Cool. Will merge.

**Pull request merged**